CHANGELOG.md

```diff
@@ -20,6 +20,10 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [4.3.1] - 2019-12-23
 
+### Added
+
+- Configure routing timeout in edxapp application
```

Comment:
@lunika I didn't noticed by it seems that you should be in the unreleased section instead of `4.3.1`... :sweat_smile: